workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/GdkCodeGenerator.csproj

```diff
@@ -19,18 +19,14 @@
     <EmbeddedResource Include="Partials\Improbable.Coordinates" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.2" />
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.1" />
-    <PackageReference Include="NUnit" Version="3.10.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
-    <PackageReference Include="NunitXml.TestLogger" Version="1.1.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Mono.Options" Version="5.3.0.1" />
```

Comment:
do we need this new dependency?

Reply:
I'm baffled because this was definitely used before - it may have been bundled in the old `CodeGenerator` dependency 🤔